### PR TITLE
Added stage for GCC ELF enclaves on windows

### LINKED ISSUE #1190: Windows CI should also test GCC ELF enclaves
Currently, Windows CI tests only use Clang ELF enclaves and this is a shortcoming. This should be extended to test GCC ELF enclaves.

### PR DESCRIPTION
**Pull request by dragoshsch**
This fixes #1190 by adding a stage for testing GCC ELF enclaves with the `Debug` configuration.

### CODE REVIEW

**Comment by dragoshsch** [comment]
cc @anitagov @ionutbalutoiu @johnkord 
bors try

**Comment by ionutbalutoiu** [comment]
@anitagov @andschwa 

If this one looks good to you, please feel free to merge it.

File: .jenkins/Jenkinsfile [hunk 1]
```diff
@@ -172,37 +178,39 @@ def win2016CrossCompile(String build_type) {
...
+         "Sim 1804 clang-7 SGX1-FLC Release" :                  { simulationTest('18.04', 'SGX1FLC', 'Release')},
+         "Sim 1804 clang-7 SGX1-FLC RelWithDebInfo" :           { simulationTest('18.04', 'SGX1FLC', 'RelWithDebInfo')},
+         "Win2016 Ubuntu1604 clang-7 Debug Linux-Elf-build" :   { win2016LinuxElfBuild('16.04', 'clang-7', 'Debug') },
+         "Win2016 Ubuntu1604 clang-7 Release Linux-Elf-build" : { win2016LinuxElfBuild('16.04', 'clang-7', 'Release') },
+         "Win2016 Ubuntu1804 clang-7 Debug Linux-Elf-build" :   { win2016LinuxElfBuild('18.04', 'clang-7', 'Debug') },
+         "Win2016 Ubuntu1804 gcc Debug Linux-Elf-build" :       { win2016LinuxElfBuild('18.04', 'gcc', 'Debug') },
+         "Win2016 Ubuntu1804 clang-7 Release Linux-Elf-build" : { win2016LinuxElfBuild('18.04', 'clang-7', 'Release') },
+         "Win2016 Debug Cross Compile" :                        { win2016CrossCompile('Debug') },
+         "Win2016 Release Cross Compile" :                      { win2016CrossCompile('Release') }
+
```
Comment (anitagov) [inline .jenkins/Jenkinsfile]: Sorry -- did not see this before. Please add a newline at the end of the file.

Comment (ionutbalutoiu) [reply .jenkins/Jenkinsfile]: @anitagov I opened https://github.com/Microsoft/openenclave/pull/1491 to address this.

**Pull request merged**